apps/meteor/ee/client/voip/modal/DialPad/hooks/useDialPad.tsx

```diff
@@ -65,26 +86,41 @@ export const useDialPad = ({ initialValue, errorMessage }: DialPadProps): DialPa
 		closeDialModal();
 	}, [outboundClient, setError, t, value, closeDialModal]);
 
+	const onSubmit = useCallback(() => {
+		handleCallButtonClick();
+	}, [handleCallButtonClick]);
+
+	const handleOnChange = useCallback(
+		(e) => {
+			clearErrors();
+			onChange(e);
+		},
+		[clearErrors, onChange],
+	);
+
 	useEffect(() => {
 		setError('PhoneInput', { message: errorMessage });
 	}, [setError, errorMessage]);
 
 	useEffect(() => {
 		setDisabled(!value);
```

Comment:
This may not work as intended if the value is falsy. Ex.: 0

Reply:
Tested value is a `string` type... are you sure?

Reply:
It seems not. You're correct. I was thinking using numbers. 🤦